Resources/Private/Fusion/Override.fusion

```diff
@@ -0,0 +1,22 @@
+prototype(Neos.Neos:Page) {
+    googleTracking = Neos.Fusion:Join {
+        googleAnalytics = Neos.GoogleAnalytics:TrackingCode.GA {
```

Comment:
Should we shorten the object names here a bit as we now have the „google“ in the wrapper?